docs/en/observability/monitor-infra/monitor-aws-services.asciidoc

```diff
@@ -1,17 +1,6 @@
 [[aws-metrics]]
 = AWS metrics
 
-[NOTE]
```

Comment:
Repeating this info across all these pages just doesn't make sense to me and might be annoying to users who just want to view the reference info about which fields to use.